ospfd/ospf_vty.c

```diff
@@ -677,8 +679,16 @@ DEFUN (ospf_area_range_cost,
 	ospf_area_display_format_set(ospf, ospf_area_get(ospf, area_id),
 				     format);
 
-	cost = strtoul(argv[idx_cost]->arg, NULL, 10);
-	ospf_area_range_cost_set(ospf, area_id, &p, cost);
+	if (argv_find(argv, argc, "cost", &idx)) {
+		cost = strtoul(argv[idx + 1]->arg, NULL, 10);
+		ospf_area_range_cost_set(ospf, area_id, &p, cost);
+	}
+
+	idx = 4;
+	if (argv_find(argv, argc, "substitute", &idx)) {
+		str2prefix_ipv4(argv[idx + 1]->arg, &s);
+		ospf_area_range_substitute_set(ospf, area_id, &p, &s);
```

Comment:
The below code is missed.
	ospf_area_display_format_set(ospf, ospf_area_get(ospf, area_id),
				     format);

Reply:
Thanks for review!
You can check it.
It's already in this function, and if you add it, it'll repeat.